diff --git a/node_modules/react-native-webrtc/android/build.gradle b/node_modules/react-native-webrtc/android/build.gradle
index 033a226..7114ab3 100644
--- a/node_modules/react-native-webrtc/android/build.gradle
+++ b/node_modules/react-native-webrtc/android/build.gradle
@@ -31,6 +31,6 @@ android {
 
 dependencies {
     implementation "com.facebook.react:react-android:+"
-    api 'org.jitsi:webrtc:124.+'
+    api 'org.jitsi:webrtc:124.0.0'
     implementation "androidx.core:core:1.7.0"
 }
